### PR TITLE
test: websockets timeout increase and logging

### PR DESCRIPTION
**Pull request by merlin-northern**
Ticket: QA-452
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline please

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline please sugar

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline please please sugar

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline please please sweet sugar

File: testutils/util/websockets.py [hunk 1]
```diff
@@ -39,16 +42,21 @@ async def connect():
                 self.url, extra_headers=self.headers, ssl=ssl_context
             )
 
-        attempts = 5
+        attempts = 15
+        sleep_seconds = 15
```
Comment (merlin-northern) [inline testutils/util/websockets.py:46]: @kacf the truth is: with that in here I was not ble to break it in any way. To the best of my testing this should get rid of the exception on lack of 101 being propagated.

File: testutils/util/websockets.py [hunk 2]
```diff
@@ -39,16 +42,21 @@ async def connect():
...
             except websockets.InvalidStatusCode:
                 if self.retry_connect and attempts > 0:
                     attempts -= 1
-                    time.sleep(5)
+                    logger.info(
+                        "websockets: %d retrying on InvalidStatusCode" % attempts
+                    )
+                    time.sleep(sleep_seconds)
                 else:
+                    logger.info("websockets: out of retryies on InvalidStatusCode")
```
Comment (kacf) [inline testutils/util/websockets.py:59]: Nitpick: **retries**

**Review by kacf — APPROVED** [review]

File: testutils/util/websockets.py (hunk 2 above)
Comment (merlin-northern) [reply testutils/util/websockets.py:59]: I really need to get a dictionary. thanks!

**Pull request merged**